examples/Misc/UploadMediaBundle.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Ads\GoogleAds\Examples\Misc;
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+use GetOpt\GetOpt;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentNames;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentParser;
+use Google\Ads\GoogleAds\Lib\V2\GoogleAdsClient;
+use Google\Ads\GoogleAds\Lib\V2\GoogleAdsClientBuilder;
+use Google\Ads\GoogleAds\Lib\V2\GoogleAdsException;
+use Google\Ads\GoogleAds\Lib\OAuth2TokenBuilder;
+use Google\Ads\GoogleAds\V2\Enums\MediaTypeEnum\MediaType;
+use Google\Ads\GoogleAds\V2\Errors\GoogleAdsError;
+use Google\Ads\GoogleAds\V2\Resources\MediaBundle;
+use Google\Ads\GoogleAds\V2\Resources\MediaFile;
+use Google\Ads\GoogleAds\V2\Services\MediaFileOperation;
+use Google\ApiCore\ApiException;
+use Google\Protobuf\StringValue;
+
+/** This example uploads an HTML5 zip file as a media bundle. */
+class UploadMediaBundle
+{
+    const CUSTOMER_ID = 'INSERT_CUSTOMER_ID_HERE';
+    const BUNDLE_URL = 'https://goo.gl/9Y7qI2';
```

Comment:
Is there any reason this should be defined at the class level instead of in the method?

Reply:
It is used only once in the code so it could be used directly in the method as you suggest.

I believe it is better to have it at class level though because it is a constant, it is more readable and it is consistent with similar code examples like [UploadImageAsset](https://github.com/googleads/google-ads-php/pull/254) and [UploadImage](https://github.com/googleads/google-ads-php/blob/master/examples/Misc/UploadImage.php).

WDYT?

Reply:
As it's not supposed to change frequently, I think it's fine to be constant here.
If we want to be more accurate, we could make it private const (probably supported since PHP 7?)
But that it's not a big deal to me.

Reply:
@devchas if no further concerns regarding this on your end, I will consider this comment closed and will open an issue to change for private constants across all examples.